lua/renamer/init.lua

```diff
@@ -375,16 +375,14 @@ function renamer._lsp_rename(word, pos)
             if resp.documentChanges then
                 changes = {}
                 for _, change in ipairs(resp.documentChanges) do
-                    changes[change.textDocument.uri] = {
-                        change.edits,
-                    }
+                    changes[change.textDocument.uri] = change.edits
```

Comment:
Are you sure about this one? I was under the impression it should be a table of tables :thinking:

Reply:
From my testing with jdtls, here is what resp.documentChanges looks like:
```
{ {
    edits = { {
        newText = "test = new Idk();\n    boolean res = ((Int) test",
        range = {
          end = {
            character = 26,
            line = 6
          },
          start = {
            character = 11,
            line = 5
          }
        }
      } },
    textDocument = {
      uri = "file:///home/axie/dev/java/random-project/src/hello/Hello.java"
    }
  } }
```
As can be seen here, for each change in the ipairs table, change.edits is already a table of tables. This changes successfully corrects `:copen` which didn't seem to be working before (shadowed by the other bug in my commit). Otherwise, having that extra table results in `Error executing vim.schedule lua callback: ...ite/pack/packer/start/renamer.nvim/lua/renamer/utils.lua:61: attempt to index
 field 'range' (a nil value)`

Here is it working with all the changes for jdtls. Not sure about other LSP's tho.
![image](https://user-images.githubusercontent.com/62098008/142825364-733b46a4-e93f-4189-b11a-bfbd0226cfe6.png)

Also, I've just noticed that jdtls does 1-indexing rather than 0-indexing for the row values :pain:

Reply:
Sure let me do that real quick :))

Reply:
Much obliged! Thanks for all the effort :grin: